lib/rules/require-curly-braces.js

```diff
@@ -97,14 +91,12 @@ module.exports.prototype = {
 
         if (typeIndex['if'] || typeIndex['else']) {
             file.iterateNodesByType('IfStatement', function(node) {
-                if (typeIndex.if && node.consequent.type !== 'BlockStatement') {
+                if (typeIndex['if'] && isNotABlockStatement(node.consequent)) {
                     addError('If', node);
                 }
-                if (typeIndex['else'] && node.alternate &&
-                    node.alternate.type !== 'BlockStatement' &&
-                    node.alternate.type !== 'IfStatement'
-                ) {
-                    addError('Else', node);
+                if (typeIndex['else'] && isNotABlockStatement(node.alternate) &&
+                    node.alternate.type !== 'IfStatement') {
```

Comment:
` && node.alternate.type !== 'IfStatement'` is kept b/c otherwise some of the tests regarding presets failed

```
modules/string-checker presets preset crockford should not report any errors from the sam
ple file:
     AssertionError: false == true
```

Not sure if it means we need to add that to disallow